Update make-calls.rst (#6232)
mattermost/docs: source/channels/make-calls.rst

Mattermost Cloud users can start calling right out of the box. For Mattermost self-hosted deployments, System Admins need to enable the plugin and adjust configurations `in the System Console </configure/plugins-configuration-settings.html#calls>`_.

Start a call
------------

.. tabs::

  .. tab:: Desktop

>     To start a call, select **Start call** in the channel or message header. When you start a call in a channel, you're muted by default. In a direct or group message you're unmuted by default.

Reviewer: For completeness I should mention that `/call start` or `/call join` slash commands can be used as well. Or even easier through the keyboard shortcut. That said, I am okay if we only want to keep the most common way here.

  .. tab:: Mobile
     

>      To start a call, select the three horizontal dots at the top right-corner of your device. Then select **Start Call**. The call starts in speaker mode by default.

Reviewer 2: You can also start a call by clicking the Channel Info menu (the channel title) and selecting Start Call from there.

>      To start a call, select the three horizontal dots at the top right-corner of your device. Then select **Start Call**. The call starts in speaker mode by default.

Author: ```suggestion
     To start a call, select the channel info menu. Then select **Start Call**. The call starts in speaker mode by default.
```

Any active team member in the channel can join a call, whether it's a public or private channel. If someone from outside of the organization wants to join a call, you'll need to provide them with a guest account, and add them to the channel. Users who are archived or not registered can't join a call.

You can share a call's link to use in a meeting request or share with other team mates. The link is unique to each channel, and contains the channel's ID, so it doesn't change between calls. Use the ``/call link`` slash command to generate a shareable link.

The call link is valid for long as the channel is active. When a channel is archived or deleted the link will become invalid.

Emojis
------

You can use emojis to react during a call.

.. tabs::

  .. tab:: Desktop

>     To do this, expand the call window using the arrows in the top-right of the call widget. From there, select the emoji icon to access frequently-used emojis or select additional emojis from the emoji picker.

Reviewer 3: is the "to do this" necessary? What are your thoughts on starting each sentence with a verb instead?

>     To do this, expand the call window using the arrows in the top-right of the call widget. From there, select the emoji icon to access frequently-used emojis or select additional emojis from the emoji picker.

Author: ```suggestion
    Expand the call window using the arrows in the top-right of the call widget. From there, select the emoji icon to access frequently-used emojis or select additional emojis from the emoji picker.
```

  .. tab:: Mobile
  

>     To do this, expand the call window using the arrows in the top-right of the active call banner. From there, select **React**.

Author: ```suggestion
    Expand the call window using the arrows in the top-right of the active call banner. From there, select **React**.
```

      
Chat in a call
--------------

> You can create a chat thread during a call, which is associated with that call. 

Author: ```suggestion
A chat thread is created automatically for every new call.
```

> You can create a chat thread during a call, which is associated with that call. 

Reviewer: A chat thread is created automatically for every new call. What the user can do is opening it up from the expanded window and interact same as on Mattermost.

.. tabs::

  .. tab:: Desktop

    To do this, expand the call window using the arrows in the top-right of the call widget. From there, select the emoji icon to access frequently-used emojis or select additional emojis from the emoji picker.

  .. tab:: Mobile
  
    To do this, expand the call window using the arrows in the top-right of the active call banner. The select **More > Call Thread**.
    
Record a call
-------------

> From Mattermost v7.7 if you're the host of a meeting, you can record the call.

Reviewer 3: Does it make sense to mention that call recording isn't supported via mobile yet/at all?

Reviewer: Managing recordings (starting/stopping) is supported from mobile as well. Unless I am misunderstanding the concern.

Reviewer: @justinegeffen It should be under the dot menu (More option) in the call screen:

![image](https://user-images.githubusercontent.com/1832946/220740267-5bfaa20d-dc8b-4149-bbc0-c818d7be709f.png)

Author: I don't have that in my version - this is from a direct message call. 

![Screenshot 2023-02-22 at 21 47 48](https://user-images.githubusercontent.com/4223530/220742351-49e5c3a4-9a40-4bab-870f-66d188c6be42.png)

Author: Using iOS 16

Reviewer: @justinegeffen Are you trying on a recordings enabled server? (e.g. https://calls.test.mattermost.cloud/). Nothing will show unless the feature is enabled and active.

Author: Oh good question. This is on community, staff team. Not sure if we have it enabled there?

Reviewer: Nope, Community doesn't have the "official" feature enabled so it won't appear there. You can give the cloud test I linked above a try.

Author: Brilliant, thank you!
